lib/sycamore/sycamore/tests/unit/test_materialize.py

```diff
@@ -331,3 +336,24 @@ def test_materialize_read(self):
             Path(tmpdir).rmdir()
             with pytest.raises(ValueError):
                 ds.take_all()
+
+
+class TestAllViaPyarrowFS(unittest.TestCase):
```

Comment:
Is it worth adding a ray test in here?

Reply:
Per discussion, not needed. There is an integration test that uses ray.